docs/releases.md

```diff
@@ -8,6 +8,17 @@ title: Release Notes
 This file contains changes and migration steps for the Graphback project. 
 Please follow individual releases for more information.
 
+# 0.17.0
```

Comment:
`1.0.0`  ?

Reply:
Hmm, we could be on v1 beta for a while and this fixes an issue which should really be fixed ASAP if someone wants to use Graphback on a stable release, that is why I did not do v1 for it.

Reply:
> Hmm, we could be on v1 beta for a while and this fixes an issue which should really be fixed ASAP if someone wants to use Graphback on a stable release, that is why I did not do v1 for it.

Okay, makes sense. I thought the idea of dropping `1.0.0` soon is after estimating that feature wise we are complete, and only bug fixes and possible performance fixes will go in. So I did not see us adding anything that could delay that release.

Reply:
/cc @wtrocki

Reply:
IMHO we have been in beta stage for a while with major changes happening. 
Instead of beta it might make sense to use 1.0.0-rc.1 (but for release notes that will be 1.0.0)

No strong opinions but I feel that 1.0.0 beta is going to send mixed messages

Reply:
We have been breaking changes in minor releases - so in this sense it was better not to have it on rc1 - breaking changes between rc* versions sounds like a nightmare. I will set it to v1 now.

Reply:
See https://github.com/aerogear/graphback/pull/2049/files

It would have been hard to maintain the templates release branch with multiple breaking changes on `templates-release-1.0.0`